src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/javadoc/writetag/InputWriteTagRecordsAndCompactCtors.java

```diff
@@ -12,7 +12,7 @@
 package com.puppycrawl.tools.checkstyle.checks.javadoc.writetag;
 
 
-public class InputWriteTagRecordsAndCompactCtors { // violation 'missing @incomplete tag.'
+public class InputWriteTagRecordsAndCompactCtors {
```

Comment:
Config is not default.
There should be no changes.

Reply:
we do update "No violation reported in case there is no javadoc."
so no violation here is ok